### PR TITLE
i18n(fr): Add `deploy/space` in French language

### PR DESCRIPTION
**Pull request by thomasbnt**
#### Description (required)

Adding `deploy/space.mdx` in French translation.

I am participating in the Hacktoberfest!

- [i18n(fr): Add deploy/space in French language](https://github.com/withastro/docs/commit/384206f0808acb0673839eec0be59acf62c06ede)
#### Related issues & labels (optional)

Nope !

### CODE REVIEW

File: src/content/docs/fr/guides/deploy/space.mdx [hunk 1]
```diff
@@ -0,0 +1,100 @@
+---
+title: Déployez votre Astro Site sur Space
+description: Comment construire un site astro sur Space.
```
Comment (dreyfus92) [inline src/content/docs/fr/guides/deploy/space.mdx:3]:
```suggestion
description: Comment construire un site Astro sur Space.
```

**Review by dreyfus92 — CHANGES_REQUESTED** [review]
Excellent travail @thomasbnt 😁.  Je vous ai laissé juste une petite suggestion

File: src/content/docs/fr/guides/deploy/space.mdx (hunk 1 above)
Comment (thomasbnt) [reply src/content/docs/fr/guides/deploy/space.mdx:3]: Owww !

**Review by pioupia — CHANGES_REQUESTED** [review]
Two little mistakes, but the rest LGTM

File: src/content/docs/fr/guides/deploy/space.mdx [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+:::
+
+
+## Configuration du projet
+
+### Site statique
+
+Apportez les modifications suivantes au fichier `Spacefile` à la racine de votre projet généré par le CLI Space.
+
+	1. Changez le moteur en "statique".
```
Comment (McFlyPartages) [inline src/content/docs/fr/guides/deploy/space.mdx:38]:
```suggestion
	1. Changez le moteur (engine) en `static`.
```

File: src/content/docs/fr/guides/deploy/space.mdx [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+	2. Ajoutez la commande de compilation d'Astro à la liste des `commandes`.
```
Comment (McFlyPartages) [inline src/content/docs/fr/guides/deploy/space.mdx:39]:
```suggestion
	2. Ajoutez la commande de compilation d'Astro à la liste des `commands`.
```

**Review by McFlyPartages — COMMENTED** [review]
I proposed two modifs also LGTM

**Comment by thomasbnt** [comment]
Thanks @McFlyPartage, @dreyfus92 @pioupia for reviews!

**Review by pioupia — APPROVED** [review]

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Great work everyone 🙌

**Pull request merged**